hack/ci/windows.ps1

```diff
@@ -191,7 +190,7 @@ Function Nuke-Everything {
         Stop-Process -name "tail" -Force -ErrorAction SilentlyContinue 2>&1 | Out-Null
 
         # Detach any VHDs
-        gwmi msvm_mountedstorageimage -namespace root/virtualization/v2 -ErrorAction SilentlyContinue | foreach-object {$_.DetachVirtualHardDisk() }
+        gwmi msvm_mountedstorageimage -namespace root/virtualization/v2 -ErrorAction SilentlyContinue | foreach-Object {$_.DetachVirtualHardDisk() }
```

Comment:
Why this change?

Reply:
Hmm I think that was when I fixed the alias, but I see `Foreach` does not have a capital here either; must've overlooked that when extracting this from the other PR